include/sbg_driver/message_wrapper.h

```diff
@@ -434,13 +451,15 @@ class MessageWrapper
     twist_stamped_message.header        = createRosHeader(ref_sbg_imu_msg.time_stamp);
     twist_stamped_message.twist.angular = ref_sbg_imu_msg.gyro;
 
-    const sbg::SbgMatrix3<float> tdcm = getTransposedDcm(ref_sbg_ekf_vel_msg);
+    sbg::SbgMatrix3f tdcm;
+	tdcm.makeTransposedDcm(ref_sbg_ekf_vel_msg);
```

Comment:
It's better to have a method to create a DCM matrix and then to transpose it rather than combining both in the same meothd :p